pkg/util/runtime/features.go

```diff
@@ -99,11 +99,11 @@ var (
 	// In each of these, keep the feature sorted by descending maturity then alphabetical
 	featureDefaults = map[Feature]bool{
 		// Beta features
+		FeatureDisableResyncOnSDKServer:         true,
 		FeatureFleetAllocateOverflow: true,
```

Comment:
huh, did this not fail `goimports`?

Reply:
used the `goimports` to format this file